src/System.Device.Gpio/System/Device/Gpio/GpioController.cs

```diff
@@ -299,5 +297,28 @@ public void Dispose()
         {
             Dispose(true);
         }
+
+        /// <summary>
+        /// Write the given pins with the given values.
```

Comment:
NIT: missing full summary with parameters and a more verbose description of what it does.

Reply:
Ok, I'll add them.